controller/work_item_type_group_blackbox_test.go

```diff
@@ -0,0 +1,62 @@
+package controller_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-wit/app/test"
+	. "github.com/fabric8-services/fabric8-wit/controller"
+	"github.com/fabric8-services/fabric8-wit/gormapplication"
+	"github.com/fabric8-services/fabric8-wit/gormsupport/cleaner"
+	"github.com/fabric8-services/fabric8-wit/gormtestsupport"
+	"github.com/fabric8-services/fabric8-wit/migration"
+	"github.com/fabric8-services/fabric8-wit/resource"
+	testsupport "github.com/fabric8-services/fabric8-wit/test"
+	almtoken "github.com/fabric8-services/fabric8-wit/token"
```

Comment:
done in https://github.com/fabric8-services/fabric8-wit/pull/1541/commits/c1cb276d949c09b342a53e552407b593bd4f0f5b